app/views/upload.scala.html

```diff
@@ -33,4 +33,13 @@ <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
         </form>
     </div>
 </div>
+<div id="progress-bar" class="govuk-grid-row" style="visibility: hidden">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">@Messages("fileProgress.header")</h1>
+        <p class="govuk-body">@Messages("fileProgress.title")</p>
+        <div>
+            <progress class="progress-bar" value="" max="50"></progress>
```

Comment:
Little bit confusing having an id and class with the same value: progress-bar. I would rename the class something like: progress-display, perhaps?